cli/tests/pcluster/cli/test_build_image.py

```diff
@@ -117,6 +129,178 @@ def test_nodejs_wrong_version_error(self, mocker, test_datadir):
             self.run_build_image_command(test_datadir)
         assert_that(exc_info.value.data.get("message")).matches("requires Node.js version >=")
 
+    def test_get_cleanup_role_name(self):
+        fake_account = "123456789012"
+        role = get_cleanup_role_name(fake_account)
+        assert_that(role).starts_with("PClusterBuildImageCleanupRole-")
```

Comment:
[Test Fragility] Why asserting the prefix and suffix of the name rather than the full name? You have an account id so the hashing is deterministic. In that way the test will be more robust!